src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,13 @@
+@mixin defaultCenter {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
I would never guess that `defaultCenter` mixin adds display flex
I don't think this mixin is really helpful
I'd better duplicate the code